protocols/v2/framing-sv2/src/framing.rs

```diff
@@ -83,16 +81,18 @@ impl<T: Serialize + GetSize, B: AsMut<[u8]> + AsRef<[u8]>> Sv2Frame<T, B> {
     /// already serialized payload. If the frame has not yet been
     /// serialized, this function should never be used (it will panic).
     pub fn payload(&mut self) -> Option<&mut [u8]> {
```

Comment:
shouldn't we rename this method?

on this commit we are sort of establishing a convention where the `Payload` variant signals that `Sv2Frame` has not yet been serialized... so calling a method called `payload` for a `Sv2Frame::Payload` and getting a `None` is extremely counter intuitive!

I feel we would create a more sane user experience if this method was called `raw` or `serialized`